### ISSUE TITLE
CeilingDiffuser inside convection model sets ACH to zero during sizing

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#6817]
Issue overview
--------------
User noted a very large difference between zone sizing peak load and the loads seen during a sizing period simulation with ideal loads. These *should* be very close to the same value. With the default TARP inside convection, they are, but with CeilingDiffuser convection they differ significantly, on the order of 30%-50%. Code inspection reveals that the CeilingDiffuser models all [set the ACH to zero during sizing](https://github.com/NREL/EnergyPlus/blob/8a7f7d6f776766877ad6790104bde49a6bc29894/src/EnergyPlus/ConvectionCoefficients.cc#L4353-L4355). Since zone sizing is calculating air mass flow rates based on the load and supply temperature, this flow rate should be available to the convection coefficient calculation.

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus v8.9.0
 - Helpdesk ticket 12897

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added EnergyPlusDevSupport\DefectFiles
 - [x] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by RKStrand** [comment]
@mjwitte The defect file that you added has ideal loads air systems in it.  How did you generate it and to what does it compare?  In other words, if I run the file that you added, which other input file will have the 30-50% differences that you mentioned in the description?

**Comment by mjwitte** [comment]
The defect file is the HVACTemplate-5ZonePurchAir example file modified to use `SurfaceConvectionAlgorithm:Inside,CeilingDiffuser`. When this file is run for sizing periods (with Chicago weather), compared the zone sizing calculated design load with the peak ????? from the csv output.  For example, for Space1-1
```
! <Zone Sizing Information>, Zone Name, Load Type, Calc Des Load {W}
SPACE1-1, Cooling, 3434.83414
```
From the csv output, the max of  SPACE1-1   IDEAL LOADS AIR SYSTEM:Zone Ideal Loads Zone Sensible Cooling Rate   [W](Hourly) is 4841.8.

The same file using TARP inside convection results in 3435W Calc Des Cooling Load and 3438W peak ideal loads sensible cooling.

**Comment by RKStrand** [comment]
@mjwitte Note that the setting of ACH to zero during sizing also occurs for the Beausoleil-Morrison convection models as they use the same logic as the CeilingDiffuser model.

**Comment by mjwitte** [comment]
@RKStrand Then it would be good to change that also if you see fit.

**Comment by RKStrand** [comment]
@mjwitte Ok, I have now run this and am seeing what you are seeing.  But, is the ACH=0 the source of the issue?  The sizing results for the ceiling diffuser convection model are essentially the same as for the TARP model (3434.834 W).  It's during the run when the ideal system is handling the load and it's no longer in sizing mod that the load is off.  It appears from running the debugger that the ACH is not zero during the actual simulation that would be producing the results for the CSV file.  So, I don't think that the ACH being set to zero is the problem.  I'll keep digging...

**Comment by mjwitte** [comment]
ACH *should be non-zero*, the ideal loads model moves air. You could make a defect file with a fancoil unit, for example, and it should have the same discrepancy between the design day sizing load and the design day simulated load.

**Comment by RKStrand** [comment]
@mjwitte Sorry--For some reason, I was trying to solve the wrong problem and thus was heading in a different direction.  What you said was totally clear so the fault in understanding was mine.  So, now that I'm on board, I have this fixed and am seeing that the sizing results for CeilingDiffuser at 4838W  and the run with ideal loads the peak is 4842W.  Will start working on the unit test, etc.  Would you like to be flagged as the tester/reviewer?

**Comment by mjwitte** [comment]
Yes, I'll review.

**Comment by mjwitte** [comment]
Closed via #7300
